nwb_conversion_tools/basedatainterface.py

```diff
@@ -17,7 +17,7 @@ def get_metadata_schema(self):
         pass
 
     @abstractmethod
-    def get_metadata(self, metadata):
+    def get_metadata(self):
```

Comment:
The usage of the downstream retrieval functions work to return a new dictionary object that the general converter then merges via deep update, so passing as a pointer to each data interface is no longer required.